Content.Shared/Damage/Components/DamageableComponent.cs

```diff
@@ -443,7 +574,10 @@ public class DamageableComponentState : ComponentState
     {
         public readonly Dictionary<DamageTypePrototype, int> DamageList;
 
-        public DamageableComponentState(Dictionary<DamageTypePrototype, int> damageList)
+        // TODO QUESTION I thought Prototypes could/should not be sent over the network? Was that just wrong or is this
```

Comment:
I have no idea how networking works

Reply:
I'm pretty sure we can't serialize prototypes yet. Paging @DrSmugleaf.